config/_default/menus/menus.en.toml

```diff
@@ -280,6 +280,27 @@ name = "Operating a Corda Node"
 parent = "corda-enterprise-4-9-corda-nodes"
 weight = 60
 
+[[corda-community-4-9]]
```

Comment:
Are these 4-9 entries correct?

Reply:
Same as in main, see: https://github.com/corda/corda-docs-portal/blob/main/config/_default/menus/menus.en.toml